pkg/agent/util/net_windows.go

```diff
@@ -896,12 +898,206 @@ func VirtualAdapterName(name string) string {
 	return fmt.Sprintf("%s (%s)", ContainerVNICPrefix, name)
 }
 
-// TODO: Implement GetInterfaceConfig for Windows
 func GetInterfaceConfig(ifName string) (*net.Interface, []*net.IPNet, []interface{}, error) {
-	return nil, nil, nil, nil
+	iface, err := net.InterfaceByName(ifName)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to get interface %s: %v", ifName, err)
+	}
+	addrs, err := GetGlobalIPNetsByName(iface)
```

Comment:
I remember @mengdie-song has renamed the function `GetGlobalIPNetsByName` in her latest update, maybe rebase and update?

Reply:
Done